### ISSUE TITLE
Invalid property

### ISSUE THREAD

**Issue by skttl** [Lecoati/LeBlender#34]
I have a LeBlender editor width two textboxes and a "Nested Content" containing some textfields and an url picker (RJP.MultiUrlPicker).

When the control is existing, and then edited, the nested content property is invalid until I make any changes in it.

Anyone else have had this problem?

**Comment by sniffdk** [comment]
Yes, actually, I've been having this issue as well, regarding the Nested Content data type.
Haven't been able to figure out where the problem originates from though.

**Comment by skttl** [comment]
I made this dirty hack in parameter.config.controller.js to make it work, if anyone has the same problem:

`283: if (!property.$valid && property.dataTypeGuid != "33b447ec-c39c-46a3-a670-4d8fa1d1cb85") {`

**Comment by sniffdk** [comment]
Yeah, did something very similar. Would be nice to get at the root problem though :)
